backend/endpoints/volunteer.py

```diff
@@ -194,6 +198,37 @@ def get_volunteers_by_filters(filters, pages=0, per_page=10000):
 def deleteVolunteer(requestjson, volunteer_id):
         updateVolunteer(requestjson, volunteer_id, delete=True)
 
+def boolconv(source):
+        if type(source) is bool:
+            if (source):
+                return 1
+            else:
+                return 0
+        else:
+            return source
+
+def volunteer_build_csv():
+    includelist = ['first_name', 'last_name', 'phone,telegram_id', 'address,zone_address',
+                   'age,offer_list', 'latitude', 'longitude', 'offer', 'received_contract']
+    si = io.StringIO()
+    # today = date.today()
+    # rnd = time.time()
+    # filename = 'volunteer_info_' + str(today) + '_' + str(rnd) + '.csv'
+    writer = csv.writer(si)
+    volunteers = [v.include_data(includelist) for v in Volunteer.objects().all()]
+
+    # write header
+    writer.writerow(volunteers[0])
+
+    # write data
+    for doc in volunteers:
+        writer.writerow([boolconv(doc[k]) for k in doc])
+
+    output = make_response(si.getvalue())
+    # output.headers["Content-Disposition"] = "attachment; filename=" + filename
+    output.headers["Content-type"] = "text/plain"
```

Comment:
```suggestion
    output.headers["Content-type"] = "text/csv"
```

Reply:
In case of text/csv a browser interprets it as attachment and i'm getting a pop-up window with save dialog. I'm not sure frontend side needs this. I need a comment from a frontend side.